### PR TITLE
Gala feedback

### PR DESCRIPTION
**Pull request by astronautthis**
@patrick-j-govalpha

### CODE REVIEW

File: module-index/module-lerp.md [hunk 1]
```diff
@@ -11,10 +11,10 @@ The Linear Interpolation Module (`lerp`) allows a governance parameter to be cha
...
+The primary purpose of the `lerp` module is to enable Governance to change a parameter gradually using only a single executive vote. The duration over which the parameter is changed, the starting value, and final value of the parameter can be chosen for each instance of the lerp module.
 
 ## Execution 
-The `lerp` contract has one method called tick(), which is permissionless and can be called by anyone. This updates the target parameter to be whatever value it should be at this moment in time. This will also finish off the `lerp` and set the parameter to the end value if the elapsed time is longer than the duration.
+The `lerp` contract has one method called tick(), which is permissionless and can be called by anyone. This updates the target parameter to be whatever value it should be at that moment. This will also finish off the `lerp` and set the parameter to the end value if the elapsed time is longer than the duration.
```
Comment (patrick-j-govalpha) [inline module-index/module-lerp.md:17]: I'd maybe rephrase the final sentence here - "If the elapsed time is longer than the duration, calling tick() will finish the lerp and set the parameter to the end value."

Presumably it will also change the done parameter - I'm not sure what to, is it "true" if lerp has ended?

File: module-index/module-lerp.md [hunk 2]
```diff
@@ -46,12 +46,12 @@ This refers to whether the given `lerp` instance is finished or not.
...
+Typically, the start and end parameters are adjusted to represent the current state and the desired state for the protocol. Hence, a longer `duration` results in the parameter being suboptimal for a longer period of time. On the other hand, more gradual changes are desirable as the protocols users can adapt more easily.
 
 
 ## Considerations
 
-When `lerp` is used to determine allocation of protocol earnings, such as setting the surplus buffer, Governance should ensure that the desired rate of increase is not larger than the expected protocol earnings. If the desired rate is too high, then there the `lerp` has no effect. 
+When `lerp` is used to determine allocation of protocol earnings, such as setting the surplus buffer, Governance should ensure that the desired rate of increase is not larger than the expected protocol earnings. If the desired rate is too high, the `lerp` has no effect.
```
Comment (patrick-j-govalpha) [inline module-index/module-lerp.md:54]: Capitals for System Surplus Buffer

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]
Mostly fine, for some reason I can't comment on it but you need to add double spacing to the end of lines for the sections at the start in blockquote so that the entries appear on separate lines

File: module-index/module-lerp.md (hunk 1 above)
Comment (astronautthis) [reply module-index/module-lerp.md]: Yes, sets done to true - Line 77 of https://github.com/makerdao/dss-lerp/blob/master/src/Lerp.sol

**Comment by patrick-j-govalpha** [comment]
I think maybe I wasn't clear about the end of line stuff.  You need to add 2 spaces to the end of line 4, 5, and 6 so that the formatting works properly. Look at the rich diff and you can see what I mean

**Comment by astronautthis** [comment]
> I think maybe I wasn't clear about the end of line stuff. You need to add 2 spaces to the end of line 4, 5, and 6 so that the formatting works properly. Look at the rich diff and you can see what I mean
Gotcha, thanks!

**Comment by patrick-j-govalpha** [comment]
It still looks weird- did you add extra blank lines? Just needs two spaces "  " at the end of each line

**Comment by astronautthis** [comment]
> 

Right, sorry I realized now how that works in markdown. Let me know if it's okay now - the only thing missing is the Technical Docs link for it (I don't think there are any other than the GitHub repo itself)

**Comment by Gala-G** [comment]
@astronautthis  - do you think you can  try resolving the conflicts? I cannot merge it like this. 

Thank you!

**Comment by astronautthis** [comment]
@Gala-G should be mergeable now. Thanks!
